### ISSUE TITLE
Support for multi-line tasks

### ISSUE THREAD

**Issue by dessalines** [mpcjanssen/simpletask-android#929]
I currently have a [PR out for todo.txt](https://github.com/todotxt/todo.txt/pull/22) that would update the spec to allow multi-line tasks, which marks them off like this:

```
(A) a single line task
---
(B) A task enclosed between lines only consisting of dashes
is treated as a +multi-line task.
---

---
(C) this is another
+multi-line
task.
---
```

Multi-line tasks would effectively turn todo.txt into a powerful notebook application. I'd love to see the parser and simpletask updated to support this.

**Comment by mpcjanssen** [comment]
Will only be added if the Todo.txt format will support it.

**Comment by dessalines** [comment]
I gotchu... unfortunately the https://github.com/todotxt/todo.txt maintainers are completely nonresponsive, and have been for over a year. The issues are piling up on that repo, with no action whatsoever. It really needs a clean fork or more implementations.... kind of like what commonmark is for markdown.

**Comment by mpcjanssen** [comment]
I personally think the todo.txt format as it is, is fine. Most of the proposed extensions proposed in that repo are trying to morph todo.txt into something it's not.
For example multiline tasks will break the one line -> one task property of todo.txt. And even with that it's still some ways from a notebook.
It's a sliding slope to reinventing org mode.

**Comment by varac** [comment]
Well, multi-line support is beeing discussed at https://github.com/todotxt/todo.txt/issues/2. Would be good to reach a conclusion there first.

**Comment by dessalines** [comment]
Its not being discussed. I issued a PR, but the maintainers haven't been active for over a year.

**Comment by smichel17** [comment]
Agreed and left my 2¢ over on the issue. Nothing wrong with multi-line in a file format that can express it, like markdown, but it's not the right fit for todo.*txt*.

**Comment by dessalines** [comment]
@mpcjanssen So [todo.txt](https://github.com/todotxt/todo.txt) is a fully dead project now. That repo has had **zero** activity for over two years. Would you be interested in helping maintain a fork?

**Comment by mpcjanssen** [comment]
@dessalines not really interested. i think todo.txt is mostly fine as it is.
